wa_examples/html_wa.json

```diff
@@ -0,0 +1,32 @@
+{
+  "id": "/wa/translation-alignment/QWxpZ25tZW50Q2h1bmtOb2RlOjE=",
+  "type": "Annotation",
+  "target": [
```

Comment:
given that the annotation will be part of an annotation keyed off of the folio urn, I've just referenced the URN for the lines here, but if we felt like an additional target to the folio urn (or image urn or canvas id or image id, as discussed in the PR body), happy to add it.

Reply:
I think this should be sufficient.